app/org/maproulette/data/DataManager.scala

```diff
@@ -442,6 +445,83 @@ class DataManager @Inject()(config: Config, db:Database)(implicit application:Ap
     }
   }
 
+  /**
+    * Gets leaderboard of top-scoring users based on task completion activity
+    * over the given period. Scoring for each completed task is based on status
+    * assigned to the task. Users are returned in descending order with top
+    * scores first; ties are broken by OSM user id with the lowest/earliest ids
+    * being ranked ahead of higher/later ids. Also included with each user are
+    * their top challenges (by amount of activity).
+    *
+    * @param start the start date
+    * @param end the end date
+    * @param limit limit the number of returned users
+    * @param offset paging, starting at 0
+    * @return Returns list of leaderboard users with scores
+    */
+  def getUserLeaderboard(start:Option[DateTime]=None, end:Option[DateTime]=None,
+                         limit:Int=Config.DEFAULT_LIST_SIZE, offset:Int=0) : List[LeaderboardUser] =
+    db.withConnection { implicit c =>
+      val parser = for {
+        userId <- long("users.id")
+        name <- str("users.name")
+        avatarURL <- str("users.avatar_url")
+        score <- int("score")
+      } yield LeaderboardUser(userId, name, avatarURL, score,
+                              this.getUserTopChallenges(userId, start, end))
+
+      SQL"""SELECT users.id, users.name, users.avatar_url, SUM(
+              CASE sa.status
+                WHEN ${Task.STATUS_FIXED} THEN 5          /* points */
```

Comment:
These values should be configurable. And then we would just retrieve what value you want from config (if none found then assume a default).